ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -417,6 +417,44 @@ default_interfaces:
         inter_domain: < true | false | Default -> True >
 ```
 
+### BGP & IP-VPN Gateway
```

Comment:
@emilarista, please help @Shivani-chourasiya update the draft schemas to cover this.

Reply:
Fix in Refactor(eos_designs): Add ipvpn_gateway in node_type_key #30 !